cmd/earthly/org_cmds.go

```diff
@@ -371,8 +374,8 @@ func (app *earthlyApp) actionOrgMemberList(cliCtx *cli.Context) error {
 func (app *earthlyApp) actionOrgMemberUpdate(cliCtx *cli.Context) error {
 	app.commandName = "orgMemberUpdate"
 
-	if cliCtx.NArg() != 2 {
-		return errors.New("member email and new permission required")
+	if cliCtx.NArg() != 1 {
```

Comment:
I would check here `< 1` and `> 1` separately, so we can say "too many arguments provided" or similar for the `> 1` case.